UnityProject/Assets/Scripts/Messages/Client/mapping/ClientRequestLoadMap.cs

```diff
@@ -19,9 +19,10 @@ public struct NetMessage : NetworkMessage
 		public Vector3 Offset; //Offset to apply 0,0 to get the position you want
 		public string Data;
 		public bool end;
-		public int MatrixID;
+		public int? MatrixID;
```

Comment:
why nullable? wouldn't we want all matricies to always have an ID? even if said ID is random?

Reply:
well null new matrix, and it gets assigned to ID by the system

Reply:
we can do nullables?

Reply:
well the C# feature that says "hey this could be potentially null"  I think it later on but we can do ? to highlight something could be null, useful value types

Reply:
yeah, I know what nullable means. I just thought we couldn't use them for some unity reason, why don't we have more nullable stuff? It would make catching bugs before they happen a lot easier

Reply:
[CanBeNull] GameObject obj yeash